conda-forge.yml

```diff
@@ -1,4 +1,5 @@
 build_platform: {osx_arm64: osx_64}
+channel_priority: flexible
```

Comment:
we should have builds for windows for pytorch right?

Reply:
Nope. They fail

Reply:
The last windows [builds](https://anaconda.org/conda-forge/pytorch-cpu/files) on conda-forge are for 1.1.0, which is too old for this recipe...

Reply:
See https://github.com/conda-forge/pytorch-cpu-feedstock/issues/32, will comment there.